.gitignore

```diff
@@ -70,6 +70,7 @@ instance/
 
 # Sphinx documentation
 docs/_build/
+docs_guidelines/_build/
```

Comment:
Is this meant to be in this PR? It looks related to something else?

Reply:
As this doesn't seem to be related to the change for this PR I think we should remove it and include it with the other sphinx guidelines work